### ISSUE TITLE
Update Strategy for MorPhoto

### ISSUE THREAD

**Issue by joewheaton** [Riverscapes/riverscapes-tools#394]

**Comment by joewheaton** [comment]
In our SOW for NASA, we said:

> The age old adage of “a picture is worth a thousand words” is nowhere truer than in the discipline of natural resource management. Collecting photos is one of the most cost effective forms of environmental monitoring. Images of change are also immensely engaging in a way that quantitative data are often not. Despite this apparent utility, it can be surprisingly difficult to capture repeat photography of the same place over time. 
>
> North Arrow Research has developed a photo monitoring software prototype that fills this apparent gap. The app captures digital photography of the same location over time. Photo time series can be shared as engaging time lapse animations to support decision making and general storytelling. 
>
> This task leverages the existing investment and will deliver a commercial grade photo monitoring app for repeat monitoring. Citizens can use this to track how their local riverscapes are changing through time with a series of photos, and resource managers can ingest this into their monitoring workflows and protocols. The app will work on mobile and desktop devices. It will also be capable of working offline and then synchronizing data when service is regained.

**Comment by joewheaton** [comment]
@philipbaileynar and @MattReimer... should we update this at all based on our recent meeting in Vancouver and discussion? Any notes?

**Comment by philipbaileynar** [comment]
Yes, but why is this ticket in the RST repo? Let's get a proper MorPhoto repo where we can all contribute and host these ideas.

**Comment by joewheaton** [comment]
> Yes, but why is this ticket in the RST repo? Let's get a proper MorPhoto repo where we can all contribute and host these ideas.

Well ... Because I don't have access to the MorPhoto repo... By all means we should have a public facing repo for MorPhoto, but it is locked somewhere in [NorthArrowResearch](https://github.com/northarrowresearch) instead of [Riverscapes](https://github.com/Riverscapes) and I don't have access to it. I think this is because @MattReimer doesn't like to show his dirty laundry and old version was demo ;)., but we need a better resting place going forward.

**Comment by MattReimer** [comment]
Nope, that's not it.... but now I need to prove it so.......

![PXL_20220531_152251638](https://user-images.githubusercontent.com/1063391/171210448-36d76df9-dda3-4873-a424-3c08d348fe36.jpg)

Real reason is that we don't have enough licenses on our NAR account and I have to keep trading them around so people can have access. It wasn't clear when we started that this repo should live in Riverscapes. It probably should move right away but I don't want to risk taking the demo down and needing to fix it.
